openff/toolkit/_tests/test_topology.py

```diff
@@ -144,6 +144,31 @@ def test_empty(self):
         assert not topology.is_periodic
         assert len(topology.constrained_atom_pairs.items()) == 0
 
+    @pytest.mark.timeout(8)
+    def test_from_molecule_multiple(self):
+        """
+        Test that add_molecule on a list of many molecules is quick.
+
+        See issue #1916
+        """
+        water = create_water()
+
+        topology = Topology()
+
+        topology.add_molecule(10_000 * [water])
```

Comment:
(not blocking) Huh, TIL

Reply:
Just a little bit more readable - I used to to think it was kinda ugly, but then I realized how frustrating counting zeros can be